EssentialCSharp.Web/wwwroot/js/site.js

```diff
@@ -171,6 +171,10 @@ const app = createApp({
                     goToPrevious();
                 }
             }
+
+            if (e.code == "KeyM" && e.ctrlKey) {

```

Comment:
Can we add a tooltip hint similar to like there is with the arrow buttons on a page, for the hamburger menu that indicates that Ctrl+M also collapses the toolbar? I'm not sure this key would be discoverable otherwise